md-commons/src/main/java/com/wipro/ats/bdre/BaseStructure.java

```diff
@@ -93,6 +89,7 @@ public final CommandLine getCommandLine(String[] params, String[][] paramStructu
             return cmd;
 
         } catch (ParseException e) {
+            LOGGER.error(e.getMessage());
             //System.err cannot be removed as it is an command line argument
             printUsage("java <main_class> ", e.getMessage(), options, System.err);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS106)